webapp/src/components/chat/chat-history/ChatHistoryItem.tsx

```diff
@@ -91,6 +94,13 @@ export const ChatHistoryItem: React.FC<ChatHistoryItemProps> = ({ message, getRe
         content = <ChatHistoryTextContent message={message} />;
     }
 
+    // TODO: hookup to backend
```

Comment:
when we leave TODOs in the app, are they:
1) for our team to follow-up on?
2) for teams taking CC as their own to later follow-up on?
3) both?

Reply:
No official guidance but our team seems to follow both. In general, if you add a TODO, you're expected to follow-up on it or change it to a comment describing the gap in development / expected behavior. Some have slipped through the cracks with the dev velocity on CC so we should probably clean those up with this migration

Reply:
> Some have slipped through the cracks with the dev velocity on CC so we should probably clean those up with this migration

I agree that a clean-up is likely in order. and with that it would be nice to have a clear distinction between TODOs left for us to follow-up on (which should usually include a Task #) vs those left for an inheriting/customer dev to take on... I can create a backlog item for this.